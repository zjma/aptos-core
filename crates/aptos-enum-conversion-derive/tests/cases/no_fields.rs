// Copyright Â© Aptos Foundation

use aptos_enum_conversion_derive::EnumConversion;

#[derive(EnumConversion)]
enum Messages {
    Test
}

fn main() {

}
