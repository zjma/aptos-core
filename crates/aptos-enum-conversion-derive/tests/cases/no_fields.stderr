error: enum variant must have exactly one field
 --> tests/cases/no_fields.rs:6:1
  |
6 | / enum Messages {
7 | |     Test
8 | | }
  | |_^
